modular_skyrat/modules/reagent_forging/code/forge_weapons.dm

```diff
@@ -112,3 +211,48 @@
 /obj/item/shield/riot/buckler/reagent_weapon/Initialize()
 	. = ..()
 	AddComponent(/datum/component/reagent_weapon)
+
+/obj/item/shield/riot/buckler/reagent_weapon/shatter(mob/living/carbon/human/owner)
+	playsound(owner, 'sound/effects/bang.ogg', 50)
+	new /obj/item/forging/complete/plate(get_turf(src))
```

Comment:
funny suggestion: have a chance of placing this in-hand instead, and having a to_chat that says "you raise your shield defensively, but find only the grip and a single plate remains!"

also, this needs a to_chat explaining that the shield just shattered, unless thats already handled somewhere else?